lib/util.js

```diff
@@ -257,6 +258,7 @@ module.exports = {
   format,
   formatWithOptions,
   getSystemErrorName,
+  getSystemErrorMap,
```

Comment:
nit: keep ASCII order
```suggestion
  getSystemErrorMap,
  getSystemErrorName,
```